internal/admin/dashboard/templates/page-oauth.html

```
@@ -0,0 +1,248 @@
+                        </button>
+                    </div>
+                </div>
+
+                <!-- Manual callback input (remote mode) -->
+                <div x-show="oauthManualProvider === provider.provider_name"
+                     style="margin-top:.6rem;padding:.6rem .75rem;border:1px solid var(--border);border-radius:6px;background:var(--bg-surface)">
+                    <p style="font-size:.8rem;margin:0 0 .4rem;opacity:.8">
+                        After authorizing, copy the full URL from the browser address bar and paste it below:
+                    </p>
+                    <div style="display:flex;gap:.4rem;align-items:center">
+                        <input type="text"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)